NuGet.config

```diff
@@ -1,13 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- Keep this file in sync with src/ProjectTemplates/test/Infrastructure/NuGet.config.in. -->
```

Comment:
Should have removed this comment. But, if everything works, I'll let comment go in. Arcade updates will probably remove it soon(ish)